lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -87,6 +87,13 @@ class GoogleCloudOutput < BufferedOutput
     # The subservice_name overrides the subservice detection, if provided.
     config_param :subservice_name, :string, :default => nil
 
+    # Valid tags require strings with only alphanumeric characters. If this
+    # config is set to true, log entries with invalid tags would be dropped and
+    # a warning would be generated in the log. If this config is set to false,
+    # we will force the tag to be valid by converting any non-string tag to
+    # string, and remove any non-utf8 or other invalid characters from it.
+    config_param :require_valid_tags, :bool, :default => true
```

Comment:
Because... technically speaking we are modifying clients' data? On a second thought, this might be better than dropping log entries. I've just changed the default value.